randovania/games/samus_returns/exporter/patch_data_factory.py

```diff
@@ -271,6 +273,9 @@ def _pickup_detail_for_target(self, detail: ExportedPickupDetails) -> dict | Non
 
         resources = get_resources_for_details(detail.original_pickup, detail.conditional_resources, detail.other_player)
 
+        if len(resources) == 1 and resources[0] == _NOTHING_RESOURCE:
+            resources = resources * len(model_names)
```

Comment:
We did try that out. We gave item_none, quantity 1 to 3 items, 2 of them being prorgressive jumps, 1 being a progressive beam.
After we collected one progressive jump the progressive beam was still at wave beam.

But again, we could've just tested wrong. 



That said, while writing this, I did realize that this will be still wrong if someone starts with e.g. HJ already.